srcStatic/VolList.cpp

```diff
@@ -12,18 +12,17 @@ void VolList::AddVolFile(std::string volPath)
 	volPaths.push_back(std::move(volPath));
 }
 
-void VolList::AddVolFilesFromDirectory(const std::string& absoluteDirectory)
+void VolList::AddVolFilesFromDirectory(const std::string& relativeSearchPath)
```

Comment:
Why did you go with the term SearchPath for the argument? I think a path could be either a directory or a filename, but we specifically want a relativeDirectory?

Reply:
You're right. I'm used to using APIs where you can specify a search pattern. In that case the pattern is not necessarily a directory name. But with the C++ filesystem API, it must be a directory name.